### ISSUE TITLE
Fail to find .GBA in scan

### ISSUE THREAD

**Issue by DanielRewcastle** [Asnivor/MedLaunch#226]
Fails to find any file with a .GBA extension even though it is the same as a .gba extension. AS such a simple fix would to be when looking at extensions make sure everything conforms to a case or just check for extensions disregarding case.

**Comment by Asnivor** [comment]
I'll take a look. That sort of thing really *should* be implemented already, so maybe there is something else funny going on.

**Comment by Asnivor** [comment]
Nope, you are actually correct. That was a bit of an oversight :)

This is now fixed in the latest dev build.
